packages/dendron-next-server/pages/vscode/calendar-view.tsx

```diff
@@ -81,19 +81,20 @@ function CalendarView({ engine, ide }: DendronProps) {
 
   const maxDots: number = 5;
   const wordsPerDot: number = 250;
-  const dailyJournalDomain = config?.journal.dailyDomain || "daily";
-  const defaultJournalName = config?.journal.name || "journal";
+  const journalDailyDomain = config?.journal.dailyDomain || "daily";
+  const journalName = config?.journal.name || "journal";
 
   // luxon token format lookup https://github.com/moment/luxon/blob/master/docs/formatting.md#table-of-tokens
-  let defaultJournalDateFormat = config?.journal.dateFormat || "y.MM.dd";
```

Comment:
instead of hardcoding, can we get the default value from `Dconfig.genDefaultConfig().journal.dateFormat`? same comments to other fields

Reply:
[`fa20956` (#1088)](https://github.com/dendronhq/dendron/pull/1088/commits/fa20956b1b0a078589402d3a7f561a646f954ba1)

I tried using `DConfig.genDefaultConfig()` from `@dendronhq/engine-server` but errord implying that the imported code fails on the frontend. We would need to move `getDefaultConfig` to `@dendronhq/common-all` or something similar.

The reason we need default in `calendar-view.tsx` is because in the first call `config` is `undefiend` and certain functions would fail because of it. It currently does accidentally not happen because `notes` and `noteActive` are at that time also `undefined` and therefore the functions won't be called.

Reply:
Solved it this way [`9a3fec3` (#1088)](https://github.com/dendronhq/dendron/pull/1088/commits/9a3fec3a25f114726ef5752af3459b7714ba0d01)